modules/ROOT/pages/admin-application-overview.adoc

```diff
@@ -1,114 +1,156 @@
+=== Change the existing pages
 
-Now, in details:
+. In the studio's Project Explorer, select the page to customize.
+ * All pages previously created using Google Web Toolkit framework have been recreated with Bonita UI Designer. They can be customized in the UI Designer. They come with the Administrator Application you have download from Bonita Studio. You will see them in the Project Explorer.
+ * All pages previously created in other, more recent technologies, have been wrapped as _custom pages_. They cannot be customized in the UI Designer. Since they cannot be imported in a .bos archive, they do not come with the Administrator Application you have downlaoded. Instead, they come as provided resources. You can find them in the "Resources" tab of the application.
+. Double-click on the page. It opens in the UI Designer. You can start editing the page.
+. Save it when you are done. Save it as another page if you need to keep Bonita's original page for further use. 
 
-* BPM menu
- ** _Monitoring_: same as in Bonita Portal.
- ** _Processes_:
-  *** Process list: recreated with the UI Designer. It brings more clarity to the unresolved-resolved-enabled lifecycle of the processes.
-  *** Process details: same as in Bonita Portal. For Community, monitoring information has been added to display what was previously in the third panel of the process list.
- ** _Cases_:
-  *** Case list: same as in Bonita Portal.
-  *** Case details: recreated with the UI Designer. It brings more clarity in the information layout and the edition of process variables.
- ** _Tasks_:
-  *** Task list: recreated with the UI Designer. It brings more clarity about what is displayed (all types of flow nodes or only human tasks), brings consistency in the page layout with the horizontal tabs, with the failed flownodes first as this is the main monitoring information to get a grip on.
-  *** Task details: recreated with the UI Designer. For failed flow nodes, the error management "replay" functionality has been clarified.
+Saving in the UI Designer makes the new page available in Bonita Studio.
 
-[WARNING]
+=== Change its navigation menu
+
+. Get all the application's pages available:
+ * If you have created new pages in the UI Designer and saved them, they are ready to be used in an application menu. 
+ * If you have created new pages in another IDE, go to the *Resources* menu option of the Administrator application, and Install your pages.
+   At deployment time, since these external resources are not part of the project, make sure you also deploy them (CAN THEY BE ATTACHED AS DEPENDENCIES IN THE MAVENIZED PROJECT IN 2021.2, OR ADDED TO THE GIT REPO IN ANOTHER WAY?)
```

Comment:
No they still can't
```suggestion
   At deployment time, since these external resources are not part of the project, make sure you also deploy them.
```